test/integration/rise-storage.html

```diff
@@ -354,25 +353,6 @@
           done();
         });
 
-        test("should retry storage request after rise-storage-api-error is encountered", function(done) {
```

Comment:
@stulees I think in the design we agreed that there should be no refresh when RC is not running. cheers